### PR TITLE
Incorporate (3->4) scripts

### PR DESCRIPTION
**Pull request by tlinjordet**
The bash files should run as `-x test` to verify the process reproducing itself. This is the larger of two steps regarding #13.

### CODE REVIEW

File: tmcinvdes/ligand_generation/screen_generated_ligands.py [hunk 1]
```diff
@@ -0,0 +1,625 @@
...
+        "Enriched SMILES",
+        "Coordination environment",
+    ]
+    unique_generated = get_unique_in_order(df_input[0].values.tolist())
+
+    unique_train = get_unique_in_order(df_train[0].values.tolist())
+    # Remove generated enriched SMILES that are verbatim string matches of training set instances:
+    unique_generated = [x for x in unique_generated if x not in unique_train]
+    validated_generated = validate_enriched_smiles_fun(unique_generated)
+    # No need to validate the train set of Enriched SMILES.
```
Comment (ABurnage) [inline tmcinvdes/ligand_generation/screen_generated_ligands.py]: Trained or training?

**Review by ABurnage — APPROVED** [review]
Found one small typo in a comment. The rest is good far as far as I can tell with my minimal coding skills.

File: tmcinvdes/ligand_generation/screen_generated_ligands.py (hunk 1 above)
Comment (tlinjordet) [reply tmcinvdes/ligand_generation/screen_generated_ligands.py]: Fixed.

**Comment by tlinjordet** [comment]
Made corrections after testing and discussing together in person.

**Comment by tlinjordet** [comment]
Note, this was PR #14  in partial fulfillment of issue #13.

**Pull request merged**